example/lib/main.dart

```diff
@@ -14,72 +17,311 @@ class MyApp extends StatefulWidget {
+                          primary: Colors.red.shade900,
+                        ),
+                      ),
+                      ElevatedButton(
+                        onPressed: () {
+                          stopLocation();
+                        },
+                        child: Text(
+                          StringConstants.stopLocationButton,
+                          style: TextStyle(
+                            fontSize: NumericConstants.fontSizeLocationsText,
+                            color: Colors.grey.shade300,
+                          ),
+                        ),
+                        style: ElevatedButton.styleFrom(
+                          primary: Colors.red.shade900,
+                        ),
+                      ),
+                    ],
+                  ),
+                  StreamBuilder(
+                    stream: AndroidPlugin.locationStream,
+                    builder: (context, snapshot) {
+                      return snapshot.hasData
+                          ? Container(
+                              padding: EdgeInsets.all(
+                                NumericConstants.coordinatesPadding,
+                              ),
+                              decoration: BoxDecoration(
+                                border: Border.all(
+                                  color: Colors.black,
+                                ),
+                              ),
+                              child: !_isStopped
+                                  ? Column(
+                                      children: [
+                                        Text(
+                                          snapshot.data
+                                              .toString()
+                                              .split("%")
+                                              .first,
+                                          style: TextStyle(
+                                            fontSize: NumericConstants
+                                                .coordinatesTextFontSize,
+                                          ),
+                                        ),
+                                        Text(
+                                          snapshot.data
+                                              .toString()
+                                              .split("%")
+                                              .last,
+                                          style: TextStyle(
+                                            fontSize: NumericConstants
+                                                .coordinatesTextFontSize,
+                                          ),
+                                        ),
+                                      ],
+                                    )
+                                  : Text(
+                                      snapshot.data.toString(),
+                                    ),
```

Comment:
You should use the same style for this Text, so it keeps consistency between messages

Reply:
Fixed